lib/core/form/components/widgets/form.theme.scss

```diff
@@ -75,11 +77,13 @@ ul > li > form-field > .adf-focus {
     }
 }
 
-
 /* query for Microsoft IE 11*/
-@media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
```

Comment:
please don't change indentation on line not affected by your cahnges

Reply:
ok, changes reverted